src/style.css

```diff
@@ -6,9 +6,15 @@
   box-sizing: border-box;
 }
 
```

Comment:
- [ ] In your index.js file, the code from lines 57 to 66 seems to create a new game on windows load. Creating a new game every time the page loads content defeats the whole essence of a leader board which is to keep track of the score of a PARTICULAR game. So the game should be created once and the `gameId` should be stored and used to POST sore and retrieve scores. The code that should run on every page load is the fetchScore. 

Below is the refactored code that reflects this suggestion. 

```javascript
let gameId = '6JrQ0TVJ8oB76U8R6SRc';

// Original Code:
// window.addEventListener('DOMContentLoaded', () => {
//   createGameOrPostScore(baseUrl, newGameObj).then((res) => {
//     gameId = res?.result;
//     const indexOfColon = gameId.indexOf(':');
//     const indexOfAdded = gameId.indexOf('added');
//     gameId = `${gameId.slice(indexOfColon + 1, indexOfAdded).trim()}`;
//     console.log(gameId);
//   });
// });

// Refactored Code:
// Reason for Refactoring:
// The original code creates a new game on every page load, which is not the intended behavior for a leaderboard.
// The purpose of a leaderboard is to track the scores of a particular game, not create a new game each time.
// Therefore, the game should be created only once, and the `gameId` should be stored and used for score posting and retrieval.
// The code that should run on every page load is the fetchScore function to display the scores for the particular game.

// Fetch the scores for the specific game using the stored `gameId`.
// This ensures that the leaderboard displays scores for the same game consistently.
window.addEventListener('DOMContentLoaded', () => {
  fetchScore(`${baseUrl}${gameId}/scores/`).then((res) => displayUser(res?.result));
});
```

Below is how your code should be if you take my suggestions. 
```javascript
import './style.css';

const refreshBtn = document.querySelector('.refresh--btn');
const form = document.querySelector('form');
const userContainer = document.querySelector('.score--list');
let gameId = '6JrQ0TVJ8oB76U8R6SRc';
const newGameObj = { name: 'ak_game' };
const baseUrl = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';

/* **** Functions **** */
const createGameOrPostScore = async (url, obj) => {
  try {
    const fecthOPtion = {
      method: 'POST',
      body: JSON.stringify(obj),
      headers: {
        'Content-type': 'application/json',
      },
    };
    const response = await fetch(url, fecthOPtion);
    const data = await response.json();
    console.log(data);
    return data;
  } catch (error) {
    return error;
  }
};
const fetchScore = async (url) => {
  try {
    const response = await fetch(url);
    const data = await response.json();
    console.log(data);
    return data;
  } catch (error) {
    return error;
  }
};

const displayUser = (users) => {
  userContainer.innerHTML = '';
  const userElement = users.map((user) => {
    const li = `
    <li class="score--list--item">
    <span>${user.user}:</span>
    <span>${user.score}</span>
    </li>`;
    return li;
  }).join('');
  if (users.length > 0) {
    userContainer.classList.remove('hide--score--list--border');
  } else {
    userContainer.classList.add('hide--score--list--border');
  }
  userContainer.insertAdjacentHTML('beforeend', userElement);
};

const updateBoard = (user) => {
  const li = `
  <li class="score--list--item">
  <span>${user.user}:</span>
  <span>${user.score}</span>
  </li>`;
  userContainer.insertAdjacentHTML('beforeend', li);
  userContainer.classList.remove('hide--score--list--border');
};

 /* **** Eventlisteners ***** */

// window.addEventListener('DOMContentLoaded', () => {
//   createGameOrPostScore(baseUrl, newGameObj).then((res) => {
//     gameId = res?.result;
//     const indexOfColon = gameId.indexOf(':');
//     const indexOfAdded = gameId.indexOf('added');
//     gameId = `${gameId.slice(indexOfColon + 1, indexOfAdded).trim()}`;
//     console.log(gameId);
//   });
// });

window.addEventListener('DOMContentLoaded', () => {
  fetchScore(`${baseUrl}${gameId}/scores/`).then((res) => displayUser(res?.result));
});


form.addEventListener('submit', async (e) => {
  e.preventDefault();
  
  const nameInput = form.querySelector('#name');
  const scoreInput = form.querySelector('#score');
  const user = nameInput.value;
  const score = scoreInput.value;
  const userInfo = { user, score };

  try {
    const res = await createGameOrPostScore(`${baseUrl}${gameId}/scores/`, userInfo);
    
    if (res?.result === 'Leaderboard score created correctly.') {
      updateBoard(userInfo);
    }
  } catch (error) {
    console.error('Error posting the score:', error);
  }

  nameInput.value = '';
  scoreInput.value = '';
});


refreshBtn.addEventListener('click', () => {
  fetchScore(`${baseUrl}${gameId}/scores/`).then((res) => displayUser(res?.result));
});
```

Reply:
Please remove the commented codes and console.logs